web/react/utils/utils.jsx

```diff
@@ -986,6 +986,10 @@ export function getFileUrl(filename) {
     }
     url = getWindowLocationOrigin() + '/api/v1/files/get' + url;
 
+    if (download) {
+        url += '?download=true';
```

Comment:
Might want to restrict this one to IE. (is it even really necessary?)  As ? URLs won't be cached by many caching systems.